GameConfiguration.h

```diff
@@ -18,8 +18,8 @@ struct GameConfiguration {
     bool show_announcements = false;
     bool show_keybinds = false;
     bool single_layer_view;
-    bool shade_hidden_tiles = false;
-    bool show_hidden_tiles;
+    bool shade_hidden_tiles = true;
```

Comment:
after seeing the difference, I'd actually vote for `shade_hidden_tiles` being kept as false by default..